MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1493,7 +1514,16 @@ private async Task<bool> RefreshExtras(BaseItem item, MetadataRefreshOptions opt
 
                 var tasks = newExtras.Select(i =>
                 {
-                    return RefreshMetadataForOwnedItem(i, true, new MetadataRefreshOptions(options), cancellationToken);
+                    var subOptions = new MetadataRefreshOptions(options);
+                    if (i.OwnerId != ownerId ||
+                        !i.ParentId.Equals(Guid.Empty))
```

Comment:
```suggestion
                        i.ParentId != Guid.Empty)
```
And use the `||` to the start of the newline